pkg/sources/reconciler/kafkasource/reconciler.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Is there no accompanying `reconciler_test.go`?

Reply:
Not yet, I'll do in a separated PR.